packages/amplify_storage_s3/android/src/main/kotlin/com/amazonaws/amplify/amplify_storage_s3/AmplifyStorageOperations.kt

```diff
@@ -18,11 +18,16 @@ package com.amazonaws.amplify.amplify_storage_s3
 import androidx.annotation.NonNull
 import android.os.Handler
 import android.os.Looper
+import com.amazonaws.amplify.amplify_auth_cognito.types.AmplifyFlutterValidationException
```

Comment:
Shouldn't this come from `Core` or `Storage`?

Reply:
This is concerning, how did it even work, do we have Auth interface dependency from Storage somehow?

Reply:
I was thinking that it compiled because the storage example app imported auth.